Assets/Scripts/UI/SlipTimeUi.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SlipTime;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI
+{
+    public class SlipTimeUi : MonoBehaviour
+    {
+        public SlipTimeManager slipTimeManager;
+    
+        public Text slipTimeUiText;
```

Comment:
Please captilize the `I` as it is preferred here.

Reply:
Fixed.